bridge/dd_optional_deps_autoloader.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace DDTrace\Bridge;
+// phpcs:disable Generic.Files.LineLength.TooLong
+
+/**
+ * Datadog Optional dependency PSR4 authoritative autoloader.
+ */
+class OptionalDepsAutoloader
+{
+    /**
+     * @var array
+     */
+    private static $autoloaderMapping = [
+        "DDTrace\\Integrations\\ZendFramework\V1\TraceRequest" => __DIR__ . '/../src/DDTrace/Integrations/ZendFramework/V1/TraceRequest.php',
```

Comment:
Also - we should update **tests/boostrap.php** to point to...

```php
require __DIR__ . '/../bridge/dd_wrap_autoloader.php';
``` 

...instead of the composer autoloader so that this kind of thing can be caught by the unit tests. :)

Reply:
Thanks!

Reply:
I experimented a bit with:
```
(find src -name “*.php”; find tests -name “*.php”) | xargs cat | grep “use DDTrace” | grep -v ‘DDTrace\\Tests\\’ | grep -v ‘::’ | awk ‘{print $2}’ | cut -f1 -d”;” | sort | uniq | sed -e ‘s#\\#\\\\#g’ |  xargs -n 1 -I %  php  -r ‘require_once “bridge/functions.php”; DDTrace\Bridge\dd_wrap_autoloader(); try { class_exists(“%”); } catch (\Throwable $) {}’
```
without good results. So using this in `tests/bootstrap.php` sounds really good to me!